### DISCUSSION TITLE
Add bunq gateway support

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#128] [Ideas]
https://together.bunq.com/d/2833-ideal-payment-url-genereren-via-api/2

**Internal Basecamp to-do:** https://basecamp.com/1810084/projects/18117394/todos/442058502

**Comment by remcotolsma** [comment]
Also see https://github.com/matthijsvdmeulen/bunq_pay.

**Comment by rvdsteege** [comment]
**Bastiaan** @ Jul 12, 2017
> Het lijkt mij een mooie toevoeging als je via de bunq API direct een iDeal betaallink kan laten genereren. Misschien via de request-inquiry endpoint, door bij counterparty_alias het "type" "IDEAL" mee te geven en de bank code als "value".
>
> Zo kun je bunq ook makkelijk als PSP gebruiken binnen bijvoorbeeld webshops.

**Koen** @ Jul 12, 2017
> Hey Bastiaan!
> 
> Dat klinkt inderdaad heel tof! 😄
> 
> Wij zijn echter geen PSP, en mogen dit daarom helaas niet doen. Maar PSP worden zou dit natuurlijk gewoon oplossen! Daarom zet ik dát voor je op de wishlist 📝
